_bmad-output/implementation-artifacts/stories/4-1-broadcast-message-isolation.md

```
@@ -0,0 +1,18 @@
+---
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)